### DISCUSSION TITLE
Welcome to scattertext Discussions!

### DISCUSSION THREAD

**Discussion by JasonKessler** [JasonKessler/scattertext#76] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dorait** [comment]
Hi, I am Dorai Thodla. You can call me Dorai. I am here as a learner. We are experimenting with keyword/entity extraction for topic/knowledge discovery.
